patten_a06c.js

```diff
@@ -0,0 +1,52 @@
+/*Bryan Patten
+patten_a06c.js
+INFO 2124
+Lisa Thoendel
+01/23/2020*/
+//winning numbers: 22, 42, 43, 47, 61;
+
+let lottoPicks = [];
+let powerBallPick = getLottoNumber(); 
+let winnings = 0; 
+
+/* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+while (lottoPicks.length < 5) {
+        let p = getLottoNumber();
+        lottoPicks.push(p);
+}
+/*While the length of the lottoPicks array is less than 5, a random number will be generated by the getLottoNumber
+function, that number will be stored in the variable p, then added to the lottoPicks array*/
+for (let i = 0; i < lottoPicks.length; i++) {
+    switch(getLottoNumber()) {

```

Comment:
Here you are switching based on a new random number. You already picked your 5 random values above in that while loop. Here you should cycle through the lottoPicks array and evaluate those numbers you already picked. something like:
switch(lottoPicks[i]) {...
Let me know if you have questions
-5